libs/components/packages/src/schematics/ng-add/ng-add.schematic.ts

```diff
@@ -1,70 +1,51 @@
-import { Rule, SchematicContext, chain } from '@angular-devkit/schematics';
+import { Rule, chain } from '@angular-devkit/schematics';
 import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import {
+  NodeDependencyType,
+  addPackageJsonDependency,
+} from '@schematics/angular/utility/dependencies';
 
 import fs from 'fs-extra';
-import getLatestVersion from 'latest-version';
 import path from 'path';
 
 import { addCrossventFix } from '../rules/add-crossvent-fix';
 import { applySkyuxStylesheetsToWorkspace } from '../rules/apply-skyux-stylesheets-to-workspace';
 import { installAngularCdk } from '../rules/install-angular-cdk';
-import { readRequiredFile } from '../utility/tree';
 import { getWorkspace } from '../utility/workspace';
 
-async function ensureLatestVersions(
-  context: SchematicContext,
-  dependencies: {
-    [_: string]: string;
-  },
-  packageGroup: {
-    [_: string]: string;
-  }
-): Promise<void> {
-  for (const packageName in dependencies) {
-    const version = packageGroup[packageName];
-
-    if (version) {
-      // Get the latest version from NPM.
-      const latestVersion = await getLatestVersion(packageName, { version });
-
-      if (latestVersion !== dependencies[packageName]) {
-        context.logger.info(
-          `Updated "${packageName}" from version (${dependencies[packageName]}) to (${latestVersion}).`
-        );
-
-        dependencies[packageName] = latestVersion;
-      } else {
-        context.logger.info(
-          `Skipped "${packageName}" because it is already on the latest version.`
-        );
-      }
+function installEssentialSkyUxPackages(skyuxVersion: string): Rule {
+  return async (tree) => {
+    const packageNames = [
+      '@skyux/assets',
+      '@skyux/core',
+      '@skyux/i18n',
+      '@skyux/theme',
+    ];
+
+    for (const packageName of packageNames) {
+      addPackageJsonDependency(tree, {
+        type: NodeDependencyType.Default,
+        name: packageName,
+        version: `^${skyuxVersion}`,
+        overwrite: true,
+      });
     }
-  }
+  };
 }
 
 export default function ngAdd(): Rule {
   return async (tree, context) => {
     const { workspace } = await getWorkspace(tree);
 
-    const packageJsonPath = 'package.json';
-    const packageJson = JSON.parse(readRequiredFile(tree, packageJsonPath));
-
-    const { packageGroup } = fs.readJsonSync(
-      path.resolve(__dirname, '../../../package.json')
-    )['ng-update'];
-
-    await ensureLatestVersions(context, packageJson.dependencies, packageGroup);
```

Comment:
☝️ We were finding the latest versions of _existing_ SKY UX packages when the user ran `ng add @skyux/packages`. This is strange since the consumer will have run `ng add @skyux/packages` before installing any other SKY UX package. I'm assuming this was implemented before we had the `skyux update` command for our internal teams, so I think we can safely remove it.